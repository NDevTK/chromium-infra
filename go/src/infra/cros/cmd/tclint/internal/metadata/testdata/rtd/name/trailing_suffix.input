{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/foo/baz/brr"
        }
    ]
}