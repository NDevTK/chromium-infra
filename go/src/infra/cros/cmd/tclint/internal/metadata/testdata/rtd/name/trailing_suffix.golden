[
"## Errors ##",
"RemoteTestDriver 'remoteTestDrivers/foo/baz/brr': name must be of the form 'remoteTestDrivers/{remoteTestDriver}', found trailing suffix 'baz/brr'"
]